script.js

```diff
@@ -510,3 +510,33 @@ function formValidate2() {
   submitErr2.innerHTML = '<i class="fa-sharp fa-solid fa-circle-check"></i>';
   return true;
 }
+const storeData = document.querySelector('.form');
+const firstName = document.getElementById('firstName');
+const lastName = document.getElementById('lastName');
+const fullName = document.getElementById('fullName');
+const email = document.getElementById('email');
+
+storeData.addEventListener('input', () => {
+  const formData = {
+    firstName: firstName.value,
+    lastName: lastName.value,
+    email: email.value,
+  };
```

Comment:
- [ ] Great job so far 👏, but one thing is missing, the message is not saved to the local storage. Please fix that.

#### Here is what you should do 👇️
 ```
  const formData = {
    firstName: firstName.value,
    lastName: lastName.value,
    email: email.value,
    msg: msg.value,
  };
 ```
```
  if (savedData) {
    const email = document.getElementById('email');
    const firstName = document.getElementById('firstName');
    const lastName = document.getElementById('lastName');
    const msg = document.getElementById('msg');

    email.value = savedData.email;
    firstName.value = savedData.firstName;
    lastName.value = savedData.lastName;
    msg.value = savedData.msg;
  }
```